src/storage-blob-preview/azext_storage_blob_preview/_params.py

```diff
@@ -118,6 +119,20 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     timeout_type = CLIArgumentType(
         help='Request timeout in seconds. Applies to each call to the service.', type=int
     )
+    t_delete_snapshots = self.get_sdk('_generated.models#DeleteSnapshotsOptionType',
+                                      resource_type=CUSTOM_DATA_STORAGE_BLOB)
+    delete_snapshots_type = CLIArgumentType(
+        arg_type=get_enum_type(t_delete_snapshots),
+        help='Required if the blob has associated snapshots. "only": Deletes only the blobs snapshots. '
+             '"include": Deletes the blob along with all snapshots.')
+    overwrite_type = CLIArgumentType(
+        arg_type=get_three_state_flag(),
+        help='Whether the blob to be uploaded should overwrite the current data. If True, upload_blob will '
+        'overwrite the existing data. If set to False, the operation will fail with ResourceExistsError. '
+        'The exception to the above is with Append blob types: if set to False and the data already exists, '
+        'an error will not be raised and the data will be appended to the existing blob. If set '
+        'overwrite=True, then the existing append blob will be deleted, and a new one created. '
+        'Defaults to False.')
```

Comment:
Should we change the `upload_blob` to some name that users can easily understand
Such as: `upload_blob` --> the operation of uploading blob

Reply:
updated